src/RapidezStatamicServiceProvider.php

```diff
@@ -19,14 +19,17 @@
 
 class RapidezStatamicServiceProvider extends ServiceProvider
 {
+    $this->app->singleton(StatamicDataComposer::class);
```

Comment:
Uh? This should be in a method somewhere?

Reply:
whoops